otter/metrics.py

```diff
@@ -44,54 +50,114 @@
     'deleting FROM scaling_group WHERE "tenantId" IN ({tids})')
 
 
-@defer.inlineCallbacks
+def valid_group_row(row):
+    return (
+        row.get('created_at') is not None and
+        row.get('desired') is not None and
+        row.get('status') not in ('DISABLED', 'ERROR') and
+        not row.get('deleting', False))
+
+
 def get_specific_scaling_groups(client, tenant_ids):
```

Comment:
No. Not really, I just left it in case it may be required later. Should I just delete it?